test/backend.test.ts

```diff
@@ -24,8 +23,8 @@ jest.mock(
     Platform: {
       OS: "mock"
     },
-    YellowBox: {
-      ignoreWarnings: jest.fn()
+    LogBox: {
+      ignoreLogs: jest.fn(),
```

Comment:
M: don't we need the `if (ReactNative.LogBox)` check here to call either one or the other? so tests won't be flakey

Reply:
Added a test to check the YellowBox fallback